src/Orchard.Web/Modules/Orchard.MediaLibrary/Views/MediaLibraryPicker.cshtml

```diff
@@ -34,7 +34,7 @@
     <label @if (required) { <text> class="required" </text>    }>@displayName</label>
     <div class="message message-Warning media-library-picker-message">@T("You need to save your changes.")</div>
     <div class="items media-library-picker" summary="@displayName">
-        <ul>
+        <ul class="media-items">
```

Comment:
@sebastienros instead of a css-class we could use data-name (or another data-* attribute), what is better?